Google AI.js

```diff
@@ -204,6 +205,16 @@ var testCases = [
 				"seeAlso": []
 			}
 		]
+	},
+	{
+		"type": "web",
+		"url": "https://research.google/research-areas/algorithms-and-theory/",
+		"items": "multiple"
+	},
+	{
+		"type": "web",
+		"url": "https://research.google/teams/applied-science/gas/",
+		"items": "multiple"
```

Comment:
Are the multiples now working as automatic tests? There is also an comment in the code that they might not work correctly. Anyway, you should either update the comment or delete the test cases for multiples again here.